orderable/templates/admin/orderable_change_list.html

```diff
@@ -7,6 +7,11 @@
     {
         cursor: pointer;
     }
+
+    /*Fix for grappelli*/
+    .ui-sortable {
+        float: inherit !important;
```

Comment:
I wonder -- is there a less aggressive way than `!important` to get this to work?